z3.ads

```diff
@@ -135,6 +135,11 @@ package Z3 is  --  GCOV_EXCL_LINE
                  Context : Z3.Context := Default_Context) return Int_Type;
    function Int (Expr : Expr_Type'Class) return Int_Type with
       Pre => Sort (Expr) = Sort_Int;
+
+   function Big_Int (Value   : String;
+                     Base    : Positive   := 10;
+                     Context : Z3.Context := Default_Context) return Int_Type;
```

Comment:
Maybe we should document which bases and number formats are allowed.

Reply:
Good point! Done in 6497444.